src/libfuncs/uint128.rs

```diff
@@ -280,21 +243,19 @@ pub fn build_is_zero<'ctx, 'this>(
 ) -> Result<()> {
     let arg0: Value = entry.argument(0)?.into();
 
-    let op = entry.append_operation(arith::constant(
+    let const_0 = entry.append_op_result(arith::constant(
```

Comment:
This constant can be created using the functions for const generation in the BlockExt trait

Reply:
fixed